src/test_l1_tvSettings.c

```diff
@@ -17471,6 +17471,150 @@ void test_l1_tvSettings_negative_EnableLocalContrast (void)
     UT_LOG("Out %s",__FUNCTION__);
 }
 
+/**
+ * @brief Validate GetLdimZoneShortCircuitStatus() for all positive invocation scenarios
+ *
+ * **Test Group ID:** Basic : 01@n
+ * **Test Case ID:** 227@n
+ *
+ * **Pre-Conditions:** None@n
+ *
+ * **Dependencies:** None@n
+ * **User Interaction:** None
+ *
+ * **Test Procedure:**@n
+ * | Variation / Step | Description | Test Data | Expected Result | Notes |
+ * | :-------: | ------------- | --------- | --------------- | ----- |
+ * | 01 | call TvInit() -  Initialise and get a valid instance of the TV client | void | tvERROR_NONE | Should Pass |
+ * | 02 | call GetLdimZoneShortCircuitStatus() -  Retrieve the shortCircuit Status | unsigned char *, int, int *  | tvERROR_NONE | Should Pass |
```

Comment:
pls update the comments int to unsigned int

Reply:
done

Reply:
Thanks, looks ok to me.  Approved